src/Helper/Helper_Data.php

```diff
@@ -291,6 +314,7 @@ public function get_localised_script_data( Helper_Abstract_Options $options, Hel
 				'deleteFontError'                      => esc_html__( 'A problem occurred. Reload the page and try again.', 'gravity-forms-pdf-extended' ),
 				'fontFileMissing'                      => esc_html__( '<strong>Oops!&nbsp;</strong>Font file gone missing. Please reupload it again.', 'gravity-forms-pdf-extended' ),
```

Comment:
@debbieellis7 we shouldn't include HTML directly in this string. The best practice way this is handled is to use `sprintf()` and put placeholders in place for the HTML in the string. Then when we output the text we replace the placeholders. 
 
```
$text = esc_html__( '%1$s<strong>Oops!%2$s' );
echo sprintf( $text, '<strong>', '</strong>' );
```

There's a good JS sprintf package we can use https://www.npmjs.com/package/sprintf-js